source/index.html.md

```diff
@@ -459,20 +462,24 @@ The last remaining task for our application is displaying the resource informati
 
 Now that we have a snazzy SMART app, it's time to test it.
 
-Next log back into the [code console](https://code.cerner.com/developer/smart-on-fhir/apps) and click on the app you've registered (My amazing SMART app). To launch your app through the code console click the "Begin Testing" button. The console will ask if the app you're launching requires a patient in context. Our app requires a patient, so select yes and choose a patient. Please note the millennium username and password, you'll need this credential when prompted. Finally, click launch and the console will redirect to your application.
+* First, log back into the [code console](https://code.cerner.com/developer/smart-on-fhir/apps) and click on the app you've registered (My amazing SMART app). 
+* To launch your app through the code console click the "Begin Testing" button. 
+  * If the app has a user scope, the console will ask if the app you're launching requires a patient in context. Because our app only has patient scopes, the question will be skipped and you just need to choose a patient to continue. 
+  * You can choose whether to display a demographics banner for the patient. This need_patient_banner flag will be part of the launch parameters. However, this tutorial app currently does not utilize this flag when displaying the result.  
+* Click "Next" to open a "Ready to launch" popup. <span style="color: red;">Take a note of the testing username and password</span>, you'll need this credential when prompted. 
```

Comment:
I dont think the <span> tag has any effect in setting the color to red if you take a look at the final version of this updated document. 
We could just leave it to the text. May be make it bold ?

Reply:
![Screen Shot 2020-02-03 at 11 27 10 AM](https://user-images.githubusercontent.com/45810410/73675539-335d6500-4678-11ea-9b8e-6665eb921349.png)

Seems like the deployed version on the tutorial has the color in RED. 
@xiaoningmzhang please ignore my comment.